### PR TITLE
[css-fonts-5] Move 'size-adjust' to its own section

### LINKED ISSUE #6110: [css-fonts] Initial value of `size-adjust` descriptor
Is there any reason to have a `normal` value for this descriptor? We could just use `100%` as the initial value.

### LINKED ISSUE #6112: [css-fonts] `size-adjust` should be moved out of the "Default font metrics overriding" section
The new `size-adjust` descriptor is not analogous to the previously-defined `ascent-override` and its friends, and probably shouldn't be grouped in the same section of the spec. Those descriptors *replace* metrics derived from the font with metrics computed from the used font size (as the section title says, they "override" things), while `size-adjust` applies a scale factor to all metrics (and images) derived from the font, but does not "override" them.

The draft text landed in #6108 includes `size-adjust` in the "metrics overriding" section, but then has to call out its different behavior multiple times. I suggest it would be better separated from the metrics overrides and described individually.

### PR DESCRIPTION
**Pull request by xiaochengh**
Fixes #6112. Fixes #6110.

### CODE REVIEW

**Comment by xiaochengh** [comment]
@litherum @svgeesus @jfkthame PTAL. Thanks!

File: css-fonts-5/Overview.bs [hunk 1]
```diff
@@ -139,13 +139,38 @@ Issue(5892):
...
+Value: <<percentage>>
+For: @font-face
+Initial: 100%
+</pre>
+
+The 'size-adjust' descriptor defines a scalar by which glyph outlines and metrics from this font
+are multiplied. Negative values are invalid at parse time.
+
+All metrics values of this font, including glyph advance sizes, are resolved as the given percentage
+multiplied by the used font size. When glyphs are visibly drawn, the visual size of the glyphs is
```
Comment (jfkthame) [inline css-fonts-5/Overview.bs:157]: I realize it's basically just being moved here, but this description is wrong; see #6133. But that could be fixed as a separate PR if you don't want to deal with it now.

Comment (xiaochengh) [reply css-fonts-5/Overview.bs:157]: I'd like to deal with it in a separate PR. Doing code move and modifications at the same time may easily lose track of things. (Removing `normal` is trivial enough that I'd like to do together...)

**Review by svgeesus — APPROVED** [review]
LGTM.

I agree that the move and any subsequent edits should be separate commits.

**Comment by xiaochengh** [comment]
Somehow the check never finishes... Could anyone help me merge it?

**Comment by svgeesus** [comment]
I noticed that the bot seems to be down; I have reported it.

@fantasai has also [made a large PR](https://github.com/w3c/csswg-drafts/pull/6154) which does this move (plus a _bunch_ of other edits) so I suspect, unless review turns up problems, we would be better keeping her PR.

Which has the same IPR bot problem

**Comment by fantasai** [comment]
Closing as fixed via https://github.com/w3c/csswg-drafts/pull/6154

**Pull request closed without merging**